### PR TITLE
Swift 2.2 - Xcode 7.3 

### PR DESCRIPTION
**Pull request by adamstener**

### CODE REVIEW

File: Bond/Core/ObservableArrayEvent.swift [hunk 1]
```diff
@@ -170,7 +170,7 @@ public extension ObservableArrayOperation {
       }
       
       if startIndex >= 0 {
-        let removedRange = Range(start: startIndex, end: endIndex)
+        let removedRange = Range(startIndex...endIndex)
```
Comment (srdanrasic) [inline Bond/Core/ObservableArrayEvent.swift]: We need to use the '..<' operator here.

**Comment by srdanrasic** [comment]
Thanks @adamstener! Could you just update that operator?

**Comment by adamstener** [comment]
I've got more to do, there were a few issues with tests and the TV target, will clean all that up and switch the operator.

**Comment by adamstener** [comment]
Not sure why the coverage dropped so much, could be because of the new #file operator doing something I don't understand yet.

**Comment by adamstener** [comment]
Is it possible that the drop in code coverage is simply due to the environment change?

**Comment by adamstener** [comment]
This probably shouldn't be pulled into master, maybe Bond needs a Swift-2.2 branch.

**Comment by lyptt** [comment]
Any progress on this?

**Comment by adamstener** [comment]
Not sure what else to do, with the new environment the testing coverage dropped.  I've been using a local copy of this but would love to figure out what it needs to get merged.

**Comment by srdanrasic** [comment]
Coverege is probably using different algorithms. Let's not worry about that right now. I'll merge this. Thank you @adamstener!

**Pull request merged**